packages/frontend/src/components/VisualizationConfigs/PieChartConfig/PieChartLayoutConfig.tsx

```diff
@@ -23,23 +24,25 @@ const PieChartLayoutConfig: React.FC = () => {
         },
     } = useVisualizationContext();
 
+    const allDimensions = [...dimensions, ...customDimensions];
```

Comment:
@joaoviana we already do this on pie charts :eyes:

Reply:
So this would help on the toggling... correct? 🤔